spec/index.html

```diff
@@ -816,6 +816,27 @@ <h3>Content Negotiation of RDF Datasets</h3>
       the consumer is expected to use the <a data-lt="rdf dataset">RDF dataset's</a> default graph.</p>
 
     </section>
+
+    <section id="section-dataset-quad" class="informative">
+      <h3>Dataset as a set of quads</h3>
```

Comment:
Just to be consistent with the capitalization of the other section titles.
```suggestion
      <h3>Dataset as a Set of Quads</h3>
```